docs/decisions/0013-system-taxonomy-auto-tagging.rst

```diff
@@ -0,0 +1,43 @@
+13. System-defined automatic tagging
+=====================================
+
+Context
+--------
+
+One main characteristic of the System-defined is the automatic content tagging. 
+It is necessary to implement this functionality when the associated content is created or edited.
+
+Decision
+---------
+
+Use `openedx-filters`_ to call the auto tagging function after content creation/edition.
+
+Filters
+~~~~~~~~
+
+It is necessary to create `filters`_ for each content for creation/edition: ``CourseCreation``, ``LibraryCreation``, etc.
+This filters will live on ``openedx-filters``.
+
+Pipelines
+~~~~~~~~~~
+
+After create the ``Content-side`` class for each content and taxonomy,
+create a function inside each class with the logic of the auto tagging.
+Then you need to register each class as `a pipeline`_ with the respective filter.
+
+This pipelines will live on ``openedx.features.tagging``
```

Comment:
I think this can be simpler, since it's only the code encapsulation we need to specify here:
```suggestion
Auto-tagging pipelines will live under ``openedx.features.tagging``, registered as `a pipeline`_ with the respective filter.
```